CONDUCT.md

```diff
@@ -0,0 +1,84 @@
+# Code of Conduct
+
+## Pledge
+
+Our Networks is dedicated to providing a harassment-free environment for everyone, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, religion or technical skill level.
+
+In order to reduce obstacles to participation, we have adopted an explicitly-documented code to communicate our expectations for conduct at the event.
+
+## Expected Behaviour
+
+We expect event participants or organizers to remain mindful of, and take responsibility for, their speech and behavior. This includes:
+
+* acting out of respect for others
+* actively listening to others and not dominating discussions
+* respecting physical and emotional boundaries
+
+## Unacceptable Behaviour
+
+We do not tolerate harassment of event participants or organizers in any form. Harassment includes, but is not limited to:
+
+* verbal comments that reinforce social structures of domination related to gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, religion, technical skill level
+* sexual images in public spaces
+* deliberate intimidation, stalking, following
+* harassing photography or recording
+* sustained disruption of talks or other events
+* inappropriate physical contact
+* unwelcome sexual attention
+* advocating for, or encouraging, any of the above behaviour
+
+Sexual language and imagery is not appropriate for Our Networks presentations and interactions. 
+
+Participants asked to stop any harassing behavior are expected to comply immediately. Participants violating these rules may be sanctioned or asked to leave, at the discretion of the organizers.
+
+## Scope
+
+This Code of Conduct applies both at the event and in public spaces when an individual is representing Our Networks.
+
+## Enforcement
+
+At events, Our Networks organizers will identify themselves and any incidents can be reported to them directly. Event organizers will be happy to help participants by providing escorts, contacting local law enforcement, or otherwise assisting those experiencing harassment to feel safe for the duration of an event. We value your participation.
+
+If a participant engages in harassing behaviour, the organizers may take any action they deem appropriate, including warning the offender or expulsion from events and online forums.
+
+If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact an organizer immediately.
+
+Those who wish to report a violation but don't feel comfortable talking to the organizers online or at an event can email [coc@ournetworks.ca](mailto:coc@ournetworks.ca), which is **currently** monitored by [**Dawn, @dcwalk**](http://dcwalker.ca), and will be monitored during the event on an active basis.
+
+Our [internal guidelines](./CONDUCT.md) for enforcing our Code of Conduct are viewable online.
```

Comment:
Should this point to `./CONDUCT.md#guidelines`?

Reply:
👍